ProcessMaker/Models/Screen.php

```diff
@@ -151,4 +153,14 @@ public function renderComponent()
         }
         return 'task-screen';
     }
+
+    /**
+     * PMQL field alias (created = created_at)
```

Comment:
It seems that this comment should have
"PMQL field alias (id = screens.id)"

Reply:
Fixed